prometheus/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,142 @@
+              type: string
+          example:
+            - <METRIC_TO_FETCH>: <NEW_METRIC_NAME>
+      - name: prometheus_metrics_prefix
+        description: Prefix for exposed Prometheus metrics.
+        value:
+          type: string
+          example: <PREFIX>_
+          display_default: null
+      - name: health_service_check
+        description: |
+          Send a service check reporting about the health of the Prometheus endpoint.
+          The service check is named <NAMESPACE>.prometheus.health
+        value:
+          type: boolean
+          example: true
+      - name: label_to_hostname
+        description: Override the hostname with the value of one label.
+        value:
+          type: string
+          example: <LABEL>
+      - name: label_joins
+        description: Allows targeting a metric to retrieve its label with a 1:1 mapping.
+        value:
+          example:
+            target_metric:
+              label_to_match: <MATCHED_LABEL>
+              labels_to_get:
+                - <EXTRA_LABEL_1>
+                - <EXTRA_LABEL_2>
+          type: object
+          properties:
+            - name: target_metric
+              type: object
+              properties:
+                - name: label_to_match
+                  type: string
+                - name: labels_to_get
+                  type: array
+                  items:
+                    type: string
+      - name: labels_mapper
+        description: |
+          The label mapper allows you to rename labels.
+          Format is <LABEL_TO_RENAME>: <NEW_LABEL_NAME>
+        value:
+          example:
+            flavor: origin
+          type: object
+      - name: type_overrides
+        description: |
+          Override a type in the Prometheus payload or type an untyped metric (ignored by default).
+          Supported <METRIC_TYPE> are `gauge`, `counter`, `histogram`, and `summary`.
+        value:
+          example:
+            <METRIC_NAME>: <METRIC_TYPE>
+          type: object
+      - name: send_histograms_buckets
+        description: Set send_histograms_buckets to true to send the histograms bucket.
+        value:
+          example: true
+          type: boolean
+      - name: send_monotonic_counter
+        description: Set send_monotonic_counter to true to send counters as monotonic counter.
+        value:
+          example: true
+          type: boolean
+      - name: exclude_labels
+        description: |
+          A list of labels to be excluded.
+        value:
+          example:
+            - timestamp
+          type: array
+          items:
+            type: string
+      - name: ssl_cert
+        description: |
+          If your prometheus endpoint is secured, here are the settings to configure it
+          Can either be only the path to the certificate and thus you should specify the private key
+          or it can be the path to a file containing both the certificate & the private key
```

Comment:
```suggestion
          If your Prometheus endpoint is secured, here are the settings to configure it:
          Can either be only the path to the certificate where you specify the private key
          or it can be the path to a file containing both the certificate and the private key.
```